best-practices/native/nativescript/images.md

```diff
@@ -0,0 +1,72 @@
+# Handle image in Nativescript-Angular
+
+To handle image in Nativescript-Angular sometimes looks hard and frustrating to do so.
+
+Doing some research I found this video from Alexander Ziskind:
+
+[Use a Hook to Process Resource Images | NativeScript Tutorial by Alexander Ziskind](https://www.youtube.com/watch?v=KgK_d-1lG84&t=46s)
+
+That video shows how `nativescript-images-generator-hook` plugin can make that task very easy.
+
+[Images Generator Hook | NativeScript Marketplace ](https://market.nativescript.org/plugins/nativescript-images-generator-hook)
+
+This plugin runs only in build moment. It is a build hook called to verify all images added in `App_resources\images` folder. Then its create copy from each image in small scale and save in to the respective folder,its depends witch platform you are building the app (android/ios).
+
+### How to setup:
+
+1 - Install the plugin
+`npm install nativescript-images-generator-hook --save-dev`
+
+2 - Create a folder named `images` inside `App_Resources` folder.
+3 - Add all our images in `images` folder.
+    Remember that images should be in the highest resolution as the app will use.
+4 - Rename the image file, adding this suffix `@?x`.
+    `?` will be a number from 1 to 5.
+
+Example: `imagename@3x.png`
+
+P.S.: The image name `can not` have space or uppercase. That I find out when I compile.
```

Comment:
What do you think of bolding **can not** here?

Reply:
looks much better. Fixed.